cmake/OpenCVFindFrameworks.cmake

```diff
@@ -32,6 +32,7 @@ if(WITH_OPENMP)
   if(OPENMP_FOUND)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+    ocv_include_directories(${OpenMP_CXX_INCLUDE_DIRS})
```

Comment:
Old CMake doesn't know about these variables. Need to add checks to properly handle that.

Ref: https://cmake.org/cmake/help/v3.5/module/FindOpenMP.html

Reply:
Will the variable not simply be empty in that case?

Reply:
any advice on this?